nifi-framework-api/src/main/java/org/apache/nifi/action/FlowActionReporter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.action;
+
+import java.util.Collection;
+
+/**
+ * A {@link FlowActionReporter} is responsible for reporting {@link FlowAction}s.
+ * Provides {@link FlowActionReporterConfigurationContext} to allow for configuration and initialization.
+ * Uses {@link #preDestruction()} to allow for cleanup. The default implementation does nothing.
```

Comment:
The `reportFlowActions()` method is required to be implemented, so this portion of the comment should be removed.
```suggestion
 * Uses {@link #preDestruction()} to allow for cleanup.
```